scripts/windows/launcher-migrate.cmd

```diff
@@ -86,6 +86,7 @@ move test.cmd .backup\ >nul
 move uninstall-service.cmd .backup\ >nul
 copy fiskaltrust.exe.config .backup\ >nul
 
+fiskaltrust.Launcher.exe config get > .backup\launcher-config-backup.json
```

Comment:
```suggestion
```

I think you misunderstood me here 😅 I've meant we need to remove this line `
copy fiskaltrust.exe.config .backup\ >nul` because we don't want to create a backup of this anymore but just leave it as it is.